packages/manager/apps/web-domains/public/translations/allDom/Messages_fr_FR.json

```
@@ -30,13 +30,30 @@
   "allDom_modal_step_two_warning": "Attention, les services suivants seront résiliés :",
   "allDom_modal_step_terminate": "Résilier",
   "allDom_modal_success_message": "Le service {{t0}} et le(s) domaine(s) {{t1}} ont bien été mis à jour",
+  "allDom_detail_page_domain_unavailable": "Ce domaine n'est pas disponible. Il sera automatiquement commandé à son retour dans le domaine public.",
   "allDom_modal_error_message": "Une erreur est survenue : {{t0}}",
   "allDom_page_detail_information_general_pack": "Pack {{t0}}",
-  "allDom_page_detail_information_general_extensions": "Les extensions incluses : {{t0}}",
-  "allDom_page_detail_subscribing_automatic_renew": "Renouvellement automatique prévu en",
+  "allDom_page_detail_information_general_extensions": "<strong>Les extensions incluses :</strong> {{t0}}",
+  "allDom_page_detail_subscribing_automatic_renew": "Renouvellement prévu le",
+  "allDom_page_detail_subscribing_manual_renew": "Expiration prévu le",
```

```suggestion
  "allDom_page_detail_subscribing_manual_renew": "Date d'expiration",
```
You already have this translation line 49, see if you can avoid duplication